src/RecentJobs/recentJobs.js

```diff
@@ -0,0 +1,34 @@
+import {useState, useEffect} from "react"
+import JobResultTable from "../JobResultTable/JobResultTable";
+import Title from "../Title/Title";
+
+function RecentJobs() {
+
+const [RecentJobData, setRecentJobData] = useState([])
+
+    const getRecentJobs = async () => {
+        let response = await fetch('http://localhost:8080/jobs/recent')
+        return await response.json()
+    }
+
+    useEffect(async () => {
```

Comment:
So the react error coming up in the terminal recommends a slightly diff approach than making the useEffect callback async:
```
useEffect(() => {
  // declare the async data fetching function
  const fetchData = async () => {
    // get the data from the api
    const data = await fetch('https://yourapi.com');
    // convert the data to json
    const json = await response.json();

    // set state with the result
    setData(json);
  }

  // call the function
  fetchData()
}, [])
```